cmd/util.go

```diff
@@ -24,3 +26,14 @@ func CheckErr(e error) {
 		os.Exit(1)
 	}
 }
+
+func getDefaultAuthTokenPath() (string, error) {
+	user, err := user.Current()
+	if err != nil {
+		return "", err
+	}
+
+	token := filepath.Join(user.HomeDir, ".mendersoftware", "authtoken")
```

Comment:
Would call the dir ".mender".

Reply:
Would also set perms to 700 for dir and 600 for auth file.

Reply:
fixed, @kjaskiewiczz can you review and merge? we should move forward with this

Reply:
ok, changes are requested so no go. @estenberg please review then